src/components/BuilderComponents/stories/Reviews.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import Reviews from "../Reviews";
+
+export default {
+  title: "BuilderComponents/Reviews",
+  component: Reviews,
+  argTypes: {
+    fgColor: { control: "color" },
+    bgColor: { control: "color" },
+    textColor: { control: "color" },
+  },
+  args: {
+    bgColor: "#faf6ee",
+    fgColor: "#9de2dd",
+    list: [
+      { title: "Check 1", content: "mic..." },
+      { title: "Check 2", content: "mic..." },
+    ],
+  },
+};
+
+const Template = args => <Reviews {...args} />;
```

Comment:
this `Template` declaration seems super repetitive

Reply:
This is actually the simplified way to avoid more code duplication. It's particularly useful when you have multiple stories for a component. Instead of recreating the component, you define it once and then bind new arguments to it to alter the render. See `HeaderAndMarkDownBlock.stories.js` for an example.